### ISSUE TITLE
Bug on starting a match

### ISSUE THREAD

**Issue by aagustan** [ValveSoftware/Dota2-Gameplay#7553]
### Description

Starting gold after picking the hero shown as 0, not able to see anything on the map when the match started. (no heroes, building, item, or courier)
Been trying to reconnect a few times, but still not able to see it.

### Example Match ID (and possibly Timestamp)

7059575600 - 10:30 AM (GMT+7)  March 14,2023

**Comment by justindg** [comment]
#6303

**Comment by justindg** [comment]
What's your GPU and/or ISP? Happens to me a lot as well.

**Comment by aagustan** [comment]
> What's your GPU and/or ISP? Happens to me a lot as well.

yes, same issue with that one. mine is even worse, no up arrow button in the top left.
So I had to end task the game first before can reconnect. 

GPU: GeForce GTX1070

I don't think this is related to ISP, as I am in Southeast Asia and have been using VPN to play
